csync/src/csync_private.h

```diff
@@ -207,7 +207,7 @@ __attribute__ ((packed))
 #endif
 ;
 
-void csync_file_stat_free(csync_file_stat_t *st);
+OCSYNC_EXPORT void csync_file_stat_free(csync_file_stat_t *st);
```

Comment:
In Qt there is a  Q_XXX_AUTIOTEST_EXPORT  for this use case.

I'm not saying we should do the same.